README.md

```diff
@@ -1 +1,174 @@
-# hello-react-front-end
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="happi.png" alt="logo" width="250"  height="auto" />
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+    [Video Demo](#video-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 hello-react-front-end <a name="about-project"></a>
+
+ **hello-react-front-end**  In this exercise, I will connect previous knowledge with new knowledge and use Webpack with React redux
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.react.com">React</li>
+    <li><a href="https://www.redux.com">Redux</li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Making use of Ruby on Rails
+- Making use of PostgreSQL
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
```

Comment:
@gilberthappi, Hi I am your code reviewer, You have one thing left, please kindly add the link to your back end also in your README for the frontend:

Reply:
okey thanks